src/types.ts

```diff
@@ -1,11 +1,14 @@
 import type { IFetchComponent } from "@well-known-components/http-server"
+import type { IPgComponent } from "@well-known-components/pg-component"
+import type * as authorizationMiddleware from "decentraland-crypto-middleware"
```

Comment:
is it required to import all the types? or could it work importing just: `DecentralandSignatureContext`?

Reply:
updated to import only `DecentralandSignatureContext`!